packages/cli/src/lib/pre-set.test.ts

```diff
@@ -6,9 +6,9 @@ describe('getEnvPreset', () => {
   afterEach(() => setupEnvVars('SANDBOX'));
   afterEach(teardownEnvVars);
 
-  it('should return default preset', () => {
+  it.skip('should return default preset', () => {
```

Comment:
Why is this skipped?

Reply:
I have 0 idea of how that got in there 😅

Thanks for catching that 🙌